README.md

```diff
@@ -0,0 +1,23 @@
+# Farm Directory API
+
+🐮 Serves up data about farms in the US with on-site markets
+
+# Getting Started
+
+Make sure you have `$ npm install nodemon -g` and [mongodb](https://www.mongodb.com/collateral/mongodb-3-2-whats-new?jmp=search&utm_source=google&utm_campaign={campaign}&utm_keyword=download%20mongo&utm_device=c&utm_network=g&utm_medium=cpc&utm_creative=112603790683&utm_matchtype=e&gclid=CPa3oJKmt88CFURahgod1FkLlA)
+
+Then `$ npm install`
+
+Then import the database dump `$ mongoimport --db farm-markets --collection farms --drop --file backup/export.json`
+
+Then `$ npm start` to boot the server
+
+Access the server from `http://localhost:3000/`
+
+# Available Routes
+
+- `/api` will return an array of all farms in the database.
+
+- `/api/zip/:zipcode` will return an array of all farms in that zipcode.
```

Comment:
Minor suggestion: Usually you want to design your endpoints around data resources/models, and then pass in filters via parameters. So this would be more like `/api/farms`,  `/api/farms?zip=02906` or `/api/farms/?state=ri`